webui/react/src/utils/themes.ts

```diff
@@ -0,0 +1,122 @@
+export enum Status {
```

Comment:
We've standardized on using "object as const" for enums, please bring this in-line.